test/unit/object/test_file.py

```diff
@@ -901,4 +901,50 @@ def test_get_thumbnail_representation_not_found(
         headers={'X-Rep-Hints': '[{}?dimensions={}]'.format(extension, dimensions)},
         params={'fields': 'representations'},
     )
-    assert thumb == b''
+    assert b'' == thumb
```

Comment:
assert thumb == b''

Reply:
They are equivalent and in general it's better to put a constant on the left so you don't accidentally have an assignment from a typo.

Reply:
I see your point, but according to pylint the other way around it preferred, see: https://vald-phoenix.github.io/pylint-errors/plerr/errors/basic/C0122 . Without fixing it the build will be still failing.